airflow/providers/amazon/aws/sensors/athena.py

```diff
@@ -71,8 +72,6 @@ def poke(self, context):
             return False
         return True
 
-    def get_hook(self):
+    def get_hook(self) -> AWSAthenaHook:
```

Comment:
Or you could try the following:

```suggestion
    @cached_property
    def hook(self) -> AWSAthenaHook:
```

If you do this, you don't need to have `self.hook` in the __init__

Reply:
`functools.cached_property` is Python >= 3.8 isn't it?

Reply:
Ah nvm, I see the pypi pkg `cached-property` is used instead